demisto_sdk/commands/pre_commit/pre_commit_command.py

```diff
@@ -687,7 +687,7 @@ def add_related_files(file: Path) -> Set[Path]:
         test_files = [
```

Comment:
i dont think we need to check if the "file.parent" exists on every iteration.

wdyt of 
```suggestion
test_files = []
if file.parent.exists():
    test_files = [_file for _file in file.parent.iterdir() if _file.name.endswith(test_file_suffix)]
```